assignments/objects.js

```diff
@@ -19,27 +19,88 @@ const example = {
 
 // Write your intern objects here:
 
+// 1,mmelloy0@psu.edu,Mitzi,F
+const mitzi = {
+  "id": 1,
+  "name": "Mitzi",
+  "email": "mmelloy0@psu.edu",
+  "gender": "F"
+}
+
+// 2,kdiben1@tinypic.com,Kennan,M
+const kennan = {
+  "id": 2,
+  "name": "Kennan",
+  "email": "kdiben1@tinypic.com",
+  "gender": "M",
+  "speak": `Hello, my name is Kennan!`
```

Comment:
So the idea here was to make this a method. Think of a method as a function nested inside an object. What you have here is a string. You want kennan.speak() to return that string. you don't want kennan.speak to be that string